### PR TITLE
Adding test for secret references

### PR DESCRIPTION
**Pull request by Teneroy**
**Description**

Changes proposed in this pull request:

- Added new test for custom http output
- Added new yaml file to test custom http output case

**Related issue(s)**

#15323

### CODE REVIEW

File: tests/fast-integration/telemetry-test/suite.js [hunk 1]
```diff
@@ -151,6 +151,26 @@ describe('Telemetry Operator', function() {
         });
       });
 
+      context('Custom HTTP Output', function() {
```
Comment (lindnerby) [inline tests/fast-integration/telemetry-test/suite.js:154]:
```suggestion
      context('Custom Output', function() {
```

File: tests/fast-integration/telemetry-test/suite.js [hunk 2]
```diff
@@ -151,6 +151,26 @@ describe('Telemetry Operator', function() {
...
+        const pipeline = loadTestData('logpipeline-output-custom.yaml');
+        const pipelineName = pipeline[0].metadata.name;
+
+        it(`Should create LogPipeline '${pipelineName}'`, async function() {
+          await k8sApply(pipeline);
+          await waitForLogPipelineStatusRunning(pipelineName);
+        });
+
+        it('Should push logs to the HTTP mockserver', async function() {
```
Comment (lindnerby) [inline tests/fast-integration/telemetry-test/suite.js:163]:
```suggestion
        it('Should push logs to the custom HTTP mockserver', async function() {
```

File: tests/fast-integration/telemetry-test/testdata/logpipeline-output-custom.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
```
Comment (lindnerby) [inline tests/fast-integration/telemetry-test/testdata/logpipeline-output-custom.yaml:4]:
```suggestion
  name: custom-output
```

**Review by lindnerby — CHANGES_REQUESTED** [review]

**Review by skhalash — CHANGES_REQUESTED** [review]
Could you please add a secret rotation to your new test similar to https://github.com/kyma-project/kyma/blob/main/tests/fast-integration/telemetry-test/suite.js#L145

**Review by lindnerby — APPROVED** [review]

**Review by skhalash — APPROVED** [review]

**Pull request merged**